notificationworkerlambda/src/main/scala/com/gu/notifications/worker/Harvester.scala

```diff
@@ -76,13 +74,7 @@ trait HarvesterRequestHandler extends Logging {
   }
 
   def switchStack(topics: List[_root_.models.Topic]): SqsDeliveryStack = {
-    if (topics.forall(topic => allowedTopicsForEc2Sender.contains(topic.toString))) {
-      logger.info("Switch to EC2-based sender"); 
-      ec2ServiceSet
-    } else {
-      logger.info("Switch to lambda-based sender"); 
-      lambdaServiceSet
-    }
+    lambdaServiceSet
```

Comment:
Now that we only deliver notifications to 1 stack, do you think we should keep this `switchStack` method, or could we simplify further?

Reply:
Yes, I think we could remove the `switchStack` function altogether.